NEWS.md

```diff
@@ -946,6 +946,8 @@ Command-line option changes
     ignored.) This flag is also available in non-`polly` builds (`USE_POLLY := 0`),
     but has no effect ([#18159]).
 
+  * The `-q` and `--quiet` flags have been deprecated in favor of `--banner={yes|no}` ([#23342]).
```

Comment:
This should be in the 0.7 section instead of 0.6, around line 320 in this file :)